pkg/types/resources.go

```diff
@@ -13,6 +13,15 @@ type Machine struct {
 	Status   MachineStatus   `json:"status"`
 }
 
+func (m *Machine) HasLabels(labels []string) bool {
+	for _, label := range labels {
+		if _, ok := m.Metadata.Labels[label]; ok {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
I don't think this method is doing what the name is saying it does
`HasLabels` means to me: "this machine has all these labels"
maybe something like `HasAnyLabel`?
btw little old unit test would be great